### DISCUSSION TITLE
Should TimML error when neither a Constant nor a semi-confining top is present?

### DISCUSSION THREAD

**Discussion by Huite** [timflow-org/timflow#13] [Q&A] [answered]
(After getting some more questions via email)

It is somewhat suprising to place a well, extracting water, and get positive isohypses. I think people (naively) expected a well to cause a drawdawn (relative to some zero). For the well, this works out to be a zero head at the well location, but this seems to end up confusing people a great deal. A single head well gives obviously weird results (-5.25e14), which is arguably less confusing.

Anyway, I'm wondering whether it doesn't make more sense to simply require a constant or a semi-confining top before solving a model, raising an error otherwise. Or is there any use case where you'd actually want "free-floating" elements?

**Comment by mbakker7** [comment, marked as answer]
This is a very good question with no simple answer. The Constant element (in AEM lingo often referred to as the reference point) fixes the head at one point and in one layer. The Constant is a model feature and not a physical feature (contrary to all other elements, which all represent physical features). In a very simple model, e.g., one well or one well in uniform flow, the Constant is useful in getting the correct head contours, but the modeler has to be aware that the Constant is not a physical feature. So, if the discharge of the well is doubled, then the head at the location of the Constant does not change, but the drawdown at the well will change. In fact, to get back to your question, a model with only a pumping well will result in a cone of depression with or without a constant. 

In a larger model, with other head-specified elements, the Constant may be used to regulate the flow outside the modeled area. But it is not required. 

And for a semi-confined aquifer, as you mention, no Constant can be added (this is checked by Tim). 

All in all, I don't think it should be required to add a Constant, although most modelers probably will add a Constant (if they simulate flow in a confined system). I think this is best explained with a good tutorial.

**Comment by dbrakenhoff** [comment]
I'm converting this to a discussion, and creating a new issue for the constant-tutorial.
